diff -Nur NVrec.orig/Makefile.am NVrec-20010808/Makefile.am
--- NVrec.orig/Makefile.am	Wed Aug  8 12:41:25 2001
+++ NVrec-20010808/Makefile.am	Fri Aug 31 14:22:28 2001
@@ -50,11 +50,11 @@
 # AVIfile / DivX ;-) record
 DIVXrec_SOURCES	= ${COREFILES} DIVXrec.c divxfile_core.cpp
 CXXFLAGS        = ${CFLAGS} @AVIFILE_CFLAGS@
-DIVXrec_LDADD   = @AVIFILE_LIBS@ 
+DIVXrec_LDADD   = @AVIFILE_LIBS@ -laviplay
 
 # DIVX4 record
 DIVX4rec_SOURCES  = ${COREFILES} DIVX4rec.c
-DIVX4rec_LDADD    = divx4/libdivx4file_core.a divx4/lame3.70/libmp3lame.a -lm @DIVX4LINUX_LIBS@
+DIVX4rec_LDADD    = divx4/libdivx4file_core.a /usr/lib/libmp3lame.a -lm @DIVX4LINUX_LIBS@
 
 # included the created .spec file in the tarball
 dist-hook: NVrec.spec
diff -Nur NVrec.orig/configure.in NVrec-20010808/configure.in
--- NVrec.orig/configure.in	Wed Aug  8 12:41:25 2001
+++ NVrec-20010808/configure.in	Fri Aug 31 14:21:52 2001
@@ -163,7 +163,7 @@
 	    enable_divx4linux=yes)
 if eval "test x$enable_divx4linux = xyes"; then
   AC_CHECK_LIB(divxencore, encore, HAVE_LIBDIVX4LINUX=yes, HAVE_LIBDIVX4LINUX=no, )
-  AC_CHECK_HEADER(encore2.h, :, HAVE_LIBDIVX4LINUX=no)
+  AC_CHECK_HEADER(divx/encore2.h, :, HAVE_LIBDIVX4LINUX=no)
   DIVX4LINUX_LIBS=-ldivxencore
 else
   HAVE_LIBDIVX4LINUX=no
diff -Nur NVrec.orig/divx4/Makefile.am NVrec-20010808/divx4/Makefile.am
--- NVrec.orig/divx4/Makefile.am	Wed Aug  8 12:41:25 2001
+++ NVrec-20010808/divx4/Makefile.am	Fri Aug 31 14:22:05 2001
@@ -1,11 +1,6 @@
-SUBDIRS	= lame3.70
-
 noinst_LIBRARIES = libdivx4file_core.a
 noinst_HEADERS	   = avilib.h
 CFLAGS  = ${MIN_CFLAGS}
 
 libdivx4file_core_a_SOURCES = avilib.c divx4file_core.c
-libdivx4file_core_a_LIBADD  = lame3.70/libmp3lame.a
-
-lame3.70/libmp3lame.a:
-	make -C lame3.70 libmp3lame.a
+libdivx4file_core_a_LIBADD  = /usr/lib/libmp3lame.a
diff -Nur NVrec.orig/tools/Makefile.am NVrec-20010808/tools/Makefile.am
--- NVrec.orig/tools/Makefile.am	Wed Aug  8 12:41:25 2001
+++ NVrec-20010808/tools/Makefile.am	Fri Aug 31 14:21:52 2001
@@ -17,7 +17,7 @@
 
 CXXFLAGS  = ${CFLAGS} @AVIFILE_CFLAGS@
 qt2divx_SOURCES = qt2divx.cpp
-qt2divx_LDADD	= @QUICKTIME_LIBS@ @AVIFILE_LIBS@
+qt2divx_LDADD	= @QUICKTIME_LIBS@ @AVIFILE_LIBS@ -laviplay
 
 quicktime_codec_RTJ0_so_SOURCES = libquicktime_codec_RTJ0.a
 
