diff -Nur NVrec.orig/divx4/divx4file_core.c NVrec-20010808/divx4/divx4file_core.c
--- NVrec.orig/divx4/divx4file_core.c	Wed Aug  8 12:41:25 2001
+++ NVrec-20010808/divx4/divx4file_core.c	Fri Aug 31 12:51:32 2001
@@ -24,7 +24,7 @@
 #include <sys/stat.h>
 #include <fcntl.h>
 #include "avilib.h"
-#include "encore2.h"
+#include "divx/encore2.h"
 #include "lame3.70/lame.h"
 #include "../NVrec.h"
 
diff -Nur NVrec.orig/divxfile_core.cpp NVrec-20010808/divxfile_core.cpp
--- NVrec.orig/divxfile_core.cpp	Wed Aug  8 12:41:25 2001
+++ NVrec-20010808/divxfile_core.cpp	Fri Aug 31 12:47:45 2001
@@ -25,9 +25,14 @@
 #include <videoencoder.h>
 #include <avifile.h>
 #include <aviplay.h>
+#include <fourcc.h>
+#include <creators.h>
+#include <stdio.h>
 #include <version.h>
 #include "NVrec.h"
 
+using namespace Creators;
+
 struct divxfile_core_t {
 	IAviWriteFile * avifile;
 	unsigned char * framebuffer;
@@ -160,7 +165,7 @@
 	
 	unlink(fname);
 	divxc->avifile = CreateIAviWriteFile(fname);
-	IVideoEncoder::SetExtendedAttr(codec, "BitRate", divxbitrate);
+	SetCodecAttr(*CodecInfo::match(codec, CodecInfo::Video,0),"BitRate", divxbitrate);	
 	fccHandler = codec;
 	framerate = 1e9 / (double)vrate;
 	
diff -Nur NVrec.orig/tools/qt2divx.cpp NVrec-20010808/tools/qt2divx.cpp
--- NVrec.orig/tools/qt2divx.cpp	Wed Aug  8 12:41:25 2001
+++ NVrec-20010808/tools/qt2divx.cpp	Fri Aug 31 12:50:17 2001
@@ -29,12 +29,15 @@
 
 #include <videoencoder.h>
 #include <avifile.h>
+#include <fourcc.h>
+#include <creators.h>
 #include <aviplay.h>
-#include <aviutil.h>
 #include <version.h>
 #include <quicktime/quicktime.h>
 #include <sys/time.h>
 
+using namespace Creators;
+
 quicktime_t *qtedopen(char *fname, long * count, unsigned long long * apos)
 {
 	quicktime_t *f = NULL;
@@ -310,7 +313,7 @@
 
 	avifile=CreateIAviWriteFile(outputfile);
 	
-	IVideoEncoder::SetExtendedAttr(codec, "BitRate", divxbitrate);
+	SetCodecAttr(*CodecInfo::match(codec, CodecInfo::Video,0),"BitRate", divxbitrate);	
 
 	BITMAPINFOHEADER bh;
 	memset(&bh,0,sizeof(BITMAPINFOHEADER));
